src/mlpack/methods/mean_shift/mean_shift.hpp

```diff
@@ -84,6 +84,7 @@ class MeanShift
   void Cluster(const MatType& data,
                arma::Row<size_t>& assignments,
                arma::mat& centroids,
+               bool forceConvergence = true,
```

Comment:
Could you add some documentation to the comments above the method describing what `forceConvergence` is please?

Reply:
Noted, will add it in.